### PR TITLE
I18 and bug fixes

### PR DESCRIPTION
**Pull request by Bartleby2718**
#### 1. Follow the instruction from https://developer.chrome.com/extensions/i18n#win-shortcut, copy and pasted below:
To create and use a shortcut that launches Google Chrome with a particular locale:

1 Make a copy of the Google Chrome shortcut that's already on your desktop.
2. Rename the new shortcut to match the new locale.
3. Change the shortcut's properties so that the Target field specifies the --lang and --user-data-dir flags. The target should look something like this:
`path_to_chrome.exe --lang=locale --user-data-dir=c:\locale_profile_dir`
***For me it looks like this: "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" --lang=ko --user-data-dir=c:\chrome-profile-ko***
4. Launch Google Chrome by double-clicking the shortcut.
For example, to create a shortcut that launches Google Chrome in Spanish (es), you might create a shortcut named chrome-es that has the following target:
`path_to_chrome.exe --lang=es --user-data-dir=c:\chrome-profile-es`
You can create as many shortcuts as you like, making it easy to test in multiple languages. For example:
`path_to_chrome.exe --lang=en --user-data-dir=c:\chrome-profile-en`
`path_to_chrome.exe --lang=en_GB --user-data-dir=c:\chrome-profile-en_GB`
`path_to_chrome.exe --lang=ko --user-data-dir=c:\chrome-profile-ko`

##  Other issues
- Contributing translations is easy. If you see the directory structure and the `messages.json` files, you'll know what to do.

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,29 +1,14 @@
 {
-  "name": "WebMark",
+  "name": "__MSG_extensionName__",
   "version": "0.1.0",
-  "description": "Load a random page from your bookmarks!",
+  "description": "__MSG_extensionDescription__",
```
Comment (jeongm-in) [inline public/manifest.json:4]: so we need to wrap the variables with `__` ?

File: public/manifest.json [hunk 2]
```diff
@@ -1,29 +1,14 @@
...
-      "suggested_key": {
-        "default": "Alt+S",
-        "mac": "Alt+S"
-      },
-      "description": "Save this page to bookmark"
-    },
-    "load": {
-      "suggested_key": {
-        "default": "Alt+Q",
-        "mac": "Alt+Q"
```
Comment (jeongm-in) [inline public/manifest.json:22]: Remind me to test these shortcuts out on a mac environment later

File: src/index.tsx [hunk 3]
```diff
@@ -146,41 +146,66 @@ var saveToWebmarkFolder = (url: string | undefined, title: string | undefined):
...
+        (results: chrome.bookmarks.BookmarkTreeNode[]): void => {
+            if (isInTree(url, results)) {
+                showNotice(
+                    constants.NotificationId.PageAlreadyExists,
+                    constants.PAGE_ALREADY_EXISTS,
+                );
+            }
+            else {
+                console.log('Same page not found in the folder.');
+                saveWithConfidence(webmarkFolderId, url, title);
```
Comment (jeongm-in) [inline src/index.tsx:143]: 😆

**Review by jeongm-in — APPROVED** [review]
I like how concise everything is now. Great work, just general questions. Approve for merge

File: public/manifest.json (hunk 1 above)
Comment (Bartleby2718) [reply public/manifest.json:4]: > In manifest.json and CSS files, refer to a string named messagename like this:
> `__MSG_messagename__`
> In your extension or app's JavaScript code, refer to a string named messagename like this:
> `chrome.i18n.getMessage("messagename")`

File: public/manifest.json (hunk 2 above)
Comment (Bartleby2718) [reply public/manifest.json:22]: https://github.com/jeongm-in/webmark/issues/13#issuecomment-508929700

**Review by jeongm-in — CHANGES_REQUESTED** [review]
Let's allow duplicate notifications and see how it is.

**Review by jeongm-in — APPROVED** [review]
Notifications work fine as expected, I really like how multiple duplicate notifications are displayed now.

**Comment by Bartleby2718** [comment]
@jeongm-in It has its limits, too. For example, It shows at most three notifications at a time, so the remaining notifications are `queue`d and shown when the previous notifications are gone.

**Pull request merged**